### DISCUSSION TITLE
error while installing foundry

### DISCUSSION THREAD

**Discussion by AVI-SHUKLA1110** [Cyfrin/foundry-full-course-cu#912] [Q&A]
## i am trying to install foundry on my windows via vscode and this is the error i am being thrown with .
```bash
curl -L https://foundry.paradigm.xyz | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:21 --:--:--     0
curl: (28) Failed to connect to raw.githubusercontent.com port 443 after 21055 ms: Timed out
```
# please help me

**Comment by Shiva-Sai-ssb** [comment]
Hello @AVI-SHUKLA1110 , I encountered a similar problem to yours when attempting to install Foundry on WSL as a Windows user. However, I managed to resolve it by connecting to a VPN. I recommend trying OpenVPN, and you can follow this instructional video to set up and establish a connection to the VPN.

https://youtu.be/jiPlad0SJ8c

**Reply by AVI-SHUKLA1110** [reply]
thnks man , it worked !🫱🏻‍🫲🏻🫱🏻‍🫲🏻
